src/components/Settings.vue

```diff
@@ -320,6 +320,26 @@
                           persistent-hint
                         ></v-switch>
                       </v-flex>
+                      <v-flex xs6 v-if="gateway.hassDiscovery">
+                        <v-text-field
+                          v-on="on"
+                          v-model="gateway.entityTemplate"
+                          label="Entity name template"
+                          persistent-hint
+                          hint="Template which generates entity names"
+                        ></v-text-field>
+                      </v-flex>
+                      <v-flex xs6 v-if="gateway.hassDiscovery">
+                        <div>
+                          Default: <code>%loc-%n_%ob</code> <br />-
+                          <code>%n</code>: Node Name <br />- <code>%loc</code>:
+                          Node Location <br />- <code>%pk</code>: valueId
+                          property key (uses property name if undefined) <br />-
+                          <code>%pn</code>: valueId property name <br />-
+                          <code>%ob</code>: HASS object_id <br />-
+                          <code>%lbl</code>: valueId label
```

Comment:
I would remove label from available placeholders, it could be very long in some values

Reply:
I do not agree, Label is really useful as it is better than other names for presenting short names, like
`Motion Detection Status`
`Battery Level`

an example of the longest i found! but makes sense
```json
{
  "id": "86-112-0-33",
  "nodeId": 86,
  "commandClass": 112,
  "commandClassName": "Configuration",
  "property": 33,
  "propertyName": "Sending Basic Set to nodes in Group 3 .",
  "type": "number",
  "readable": true,
  "writeable": true,
  "description": "Sending Basic Set to nodes in Group 3 .",
  "label": "Sending Basic Set to nodes in Group 3 .",
  "default": 3,
  "min": 0,
  "max": 4,
  "list": true,
  "states": [
    {
      "text": "Don’t send Basic Set.",
      "value": 0
    },
    {
      "text": "Option 1",
      "value": 1
    },
    {
      "text": "Option 2",
      "value": 2
    },
    {
      "text": "Option 3 (default)",
      "value": 3
    },
    {
      "text": "Option 4",
      "value": 4
    }
  ],
  "value": 3,
  "lastUpdate": 1608813167188,
  "nodeName": "Doorbell",
  "nodeLocation": "MetersBox"
}
```

Where typically is: 
```json

{
  "id": "87-128-0-level",
  "nodeId": 87,
  "commandClass": 128,
  "commandClassName": "Battery",
  "endpoint": 0,
  "property": "level",
  "propertyName": "level",
  "type": "number",
  "readable": true,
  "writeable": false,
  "label": "Battery level",
  "min": 0,
  "max": 100,
  "list": false,
  "value": 100,
  "lastUpdate": 1609086123492,
  "nodeName": "Button",
  "nodeLocation": "Kitchen"
}
```
in the second example, label makes more sense than level, except if we allow also Command Clases to join the list of placholders.
another example label fits the purpose:
```json
{
  "id": "97-48-0-Any",
  "nodeId": 97,
  "commandClass": 48,
  "commandClassName": "Binary Sensor",
  "endpoint": 0,
  "property": "Any",
  "propertyName": "Any",
  "type": "boolean",
  "readable": true,
  "writeable": false,
  "label": "Any",
  "ccSpecific": {
    "sensorType": 255
  },
  "list": false,
  "value": false,
  "lastUpdate": 1609088758510,
  "nodeName": "Sensor",
  "nodeLocation": "Hallway"
}
```

Reply:
Ok keep the label so :+1: